lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -1023,4 +1031,94 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
   end
 
   defp intellisense_node(_node), do: node()
+
+  defp build_identifiers(metadata) do
+    identifiers_defined = metadata.identifiers_defined
+
+    Enum.reduce_while(identifiers_defined, {:ok, %{}}, fn
+      {{:module, module}, _}, {:ok, acc} ->
+        with {^module, bin, _} <- :code.get_object_code(module),
+             {:ok, debug_info} <- beam_lib_chunks(bin, :debug_info),
+             {:ok, abstract_code} <- beam_lib_chunks(bin, :abstract_code),
+             {:ok, types} <- Code.Typespec.fetch_types(module) do
```

Comment:
I think this will work too:

```suggestion
             {:ok, types} <- Code.Typespec.fetch_types(bin) do
```

Reply:
I don't think there are other cons. The question is if you want to point to the beginning of the docs or to the signature.

Reply:
I think pointing ton the signature (`defmodule ...` or `def ...`) would be better.